tools/pyclient/LICENSE

```diff
@@ -0,0 +1,165 @@
+GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
can we somehow symlink to the overal license?

Reply:
added symlink to LICENSE